client/state/reader/conversations/schema.js

```diff
@@ -3,22 +3,14 @@
 /**
  * Internal dependencies
  */
-import {
-	CONVERSATION_FOLLOW_STATUS_FOLLOWING,
-	CONVERSATION_FOLLOW_STATUS_NOT_FOLLOWING,
-	CONVERSATION_FOLLOW_STATUS_MUTING,
-} from './follow-status';
+import { CONVERSATION_FOLLOW_STATUS } from './follow-status';
 
 /* eslint-disable quote-props */
 export const itemsSchema = {
 	type: 'object',
 	patternProperties: {
 		'^[0-9]+-[0-9]+$': {
-			enum: [
-				CONVERSATION_FOLLOW_STATUS_FOLLOWING,
-				CONVERSATION_FOLLOW_STATUS_NOT_FOLLOWING,
-				CONVERSATION_FOLLOW_STATUS_MUTING,
-			],
+			enum: Object.values( CONVERSATION_FOLLOW_STATUS ),
```

Comment:
🎉

Reply:
its like one tenth of the way towards actual types :O

Reply:
`Object.values` doesn't work in IE. I'm having trouble figuring out if we polyfill it

Reply:
Ah - good point. Let's just use lodash `values`.